src/components/table/table.styleOverrides.ts

```diff
@@ -0,0 +1,30 @@
+export const styleOverrides = {
+  overflowX: 'auto',
+  borderRadius: '8px',
+  peopleflow: {
+    '& .MuiDataGrid-cell:hover': {
+      cursor: 'pointer',
+    },
+    '.MuiDataGrid-columnHeader': {
+      backgroundColor: 'rgb(115, 135, 195)',
+      color: 'white',
+    },
+    '.MuiDataGrid-columnHeaderTitle': {
+      fontWeight: 'bolder',
+    },
+    '.MuiIconButton': {
+      color: 'white',
+    },
+    '.MuiDataGrid-columnHeaderTitleContainerContent .MuiCheckbox-root': {
+      color: 'white',
+    },
+    '.MuiDataGrid-columnHeader .MuiSvgIcon-root': {
+      color: 'white',
+    },
+    '.MuiDataGrid-cell .MuiSvgIcon-root': {
+      fill: 'rgb(115, 135, 195)',
+    },
+  },
+  skillbook: {},
+  worklight: {},
```

Comment:
Would be nice to chat about this file and how we want to override style based on theme category